Documentation update for SCP-B2B-1.0.19 and SCP-4.8.18 (#829)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/storefront-compatibility-b2b.mdx

The Storefront Compatibility B2B Package defines the following queries and mutations:

> - [`negotiableQuoteTemplates`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/queries/templates/) query - Retrieves a list of negotiable quote templates along with their details, including expiration dates.

Reviewer: ```suggestion
- [`negotiableQuoteTemplates`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/queries/templates/) query - Retrieves negotiable quote templates with details, including their expiration dates.
```

Author: Fixed

> - [`placeNegotiableQuoteOrderV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/mutations/place-order-v2/) mutation - Converts a negotiable quote into an order and returns comprehensive order details along with an errors array for consistent response handling.

Reviewer: ```suggestion
- [`placeNegotiableQuoteOrderV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/mutations/place-order-v2/) mutation - Transforms a negotiable quote into an order and returns full order details, including an errors array for standardized response handling.
```

Author: Fixed

> - [`setQuoteTemplateExpirationDate`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/mutations/set-quote-template-expiration-date/) mutation - Allows negotiable quote templates to include an expiration date using setQuoteTemplateExpirationDate.

Reviewer: ```suggestion
- [`setQuoteTemplateExpirationDate`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/negotiable-quote/mutations/set-quote-template-expiration-date/) mutation - Sets an expiration date for negotiable quote templates via the setQuoteTemplateExpirationDate function.
```

Author: Fixed

> - [`sharedRequisitionList`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/queries/shared-requisition-list/) query - Allows B2B customers to retrieve a shared Requisition List using a token, including sender details and associated items in a read-only format.

Reviewer: ```suggestion
- [`sharedRequisitionList`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/queries/shared-requisition-list/) query - Retrieves a read-only shared requisition list for B2B customers, including sender details and associated items, based on the provided token.
```

Author: Fixed

> - [`shareRequisitionListByToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/share-requisition-list-by-token/) mutation - Enables B2B customers to share a Requisition List with colleagues within the same company via a shareable link generated on the storefront.

Reviewer: ```suggestion
- [`shareRequisitionListByToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/share-requisition-list-by-token/) mutation - Generates a shareable storefront link that enables B2B customers to share a requisition list with colleagues within the same company.
```

Author: Fixed

> - [`shareRequisitionListByEmail`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/share-requisition-list-by-email/) mutation - Enables B2B customers to share a Requisition List with colleagues within the same company via email.

Reviewer: ```suggestion
- [`shareRequisitionListByEmail`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/share-requisition-list-by-email/) mutation - Sends a requisition list to colleagues within the same company via email for B2B customers.
```

Author: Fixed

> - [`importSharedRequisitionList`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/import-shared-requisition-list/) mutation - Allows recipients within the same company to import or clone a shared Requisition List using a token.

Reviewer: ```suggestion
- [`importSharedRequisitionList`](https://developer.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/import-shared-requisition-list/) mutation - Imports or clones a shared requisition list for recipients within the same company based on the provided token.
```

Author: Fixed
